src/dotnet-new3/dotnet-new3.csproj

```diff
@@ -1,7 +1,7 @@
 <Project ToolsVersion="15.0" Sdk="Microsoft.NET.Sdk">
     <PropertyGroup>
         <Description>Template Instantiation Commands for .NET Core CLI.</Description>
-        <TargetFramework>netcoreapp2.1</TargetFramework>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
```

Comment:
not sure we can take this update as part of servicing without getting the approval. Is there any way to avoid it?

Reply:
Build had the error NETSDK1138: The target framework 'netcoreapp2.1' is out of support. According to https://learn.microsoft.com/en-us/dotnet/core/tools/sdk-errors/netsdk1138, `<CheckEolTargetFramework>false</CheckEolTargetFramework>` can resolve it.